packages/lib/display/service.ts

```diff
@@ -64,7 +64,7 @@ export const getDisplay = async (displayId: string): Promise<TDisplay | null> =>
       revalidate: SERVICES_REVALIDATION_INTERVAL,
     }
   )();
-  return display ? formatDateFields(display, ZDisplay) : null;
+  return display ? formatDateFields({ ...display, status: null }, ZDisplay) : null;
```

Comment:
I think instead of this hack you need to add the `status` to `selectDisplay`, so that it's requested from the database.
With your hack the status would be always null, even when a status is set in the database.